### ISSUE TITLE
Unable to find forwarding rule for host with SNS subscription confirmation

### ISSUE THREAD

**Issue by lephuongbg** [localstack/localstack#3431]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

Send a request to the SNS subscription confirmation URL should succeed but since v0.12.0 it failed.

## Expected behavior

The subscription confirmation succeeds.

## Actual behavior

```
2021-01-06T02:52:59:INFO:localstack.services.edge: Unable to find forwarding rule for host "localstack:4566", path "/?Action=ConfirmSubscription&TopicArn=arn:aws:sns:ap-northeast-1:000000000000:data-event-bus&Token=e256767a", target header "", auth header "", data "b''"
```

# Steps to reproduce

## Command used to start LocalStack

```yaml
services:
  localstack:
    image: localstack/localstack-light:0.12.0
    environment:
      - SERVICES=sns
      - DEFAULT_REGION=ap-northeast-1
      - HOSTNAME_EXTERNAL=localstack
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

```python
requests.get(message["SubscribeURL"])
```

**Comment by anushacassum** [comment]
Yes please fix this soon :( Thanks

**Comment by whummer** [comment]
Thanks for reporting @lephuongbg @anushacassum . This should be fixed in #3437 - can you please pull the latest Docker image and give it another try? Please report here if the problem persists. Thanks
